packages/cli/src/databases/migrations/mysqldb/1681134145996-AddUserActivatedProperty.ts

```diff
@@ -0,0 +1,57 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { getTablePrefix, logMigrationEnd, logMigrationStart } from '@db/utils/migrationHelpers';
+import { userSettings } from '@/Interfaces';
+
+export class AddUserActivatedProperty1681134145996 implements MigrationInterface {
+	name = 'AddUserActivatedProperty1681134145996';
+
+	async up(queryRunner: QueryRunner): Promise<void> {
+		logMigrationStart(this.name);
+
+		const tablePrefix = getTablePrefix();
+
+		const activatedUsers: userSettings[] = await queryRunner.query(
+			`SELECT sw.userId AS id,
+				JSON_SET(COALESCE(u.settings, '{}'), '$.userActivated', true) AS settings
+			FROM ${tablePrefix}workflow_statistics AS ws
+						JOIN ${tablePrefix}shared_workflow as sw
+							ON ws.workflowId = sw.workflowId
+						JOIN ${tablePrefix}role AS r
+							ON r.id = sw.roleId
+						JOIN ${tablePrefix}user AS u
+							ON u.id = sw.userId
+			WHERE ws.name = 'production_success'
+						AND r.name = 'owner'
+						AND r.scope = 'workflow'`,
+		);
+
+		const updatedUsers = activatedUsers.map((user) =>
+			queryRunner.query(
+				`UPDATE ${tablePrefix}user SET settings = '${user.settings}' WHERE id = '${user.id}' `,
```

Comment:
Still getting errors in this part; see the attached screenshot
<img width="631" alt="Screenshot 2023-04-14 at 12 01 01" src="https://user-images.githubusercontent.com/219272/232014492-0b53d589-8087-449f-985d-73fdf1335224.png">

The `user.settings` needs to be stringified first I think.